src/environments/environment.prod.ts

```diff
@@ -4,8 +4,8 @@ import env from './.env';
 export const environment = {
   production: true,
   version: env.npm_package_version,
-  fineractPlatformTenantId: 'staging',  // For connecting to server running elsewhere update the tenant identifier
-  baseApiUrl: 'https://staging.openmf.org',  // For connecting to server running elsewhere update the base API URL
+  fineractPlatformTenantId: 'default',  // For connecting to server running elsewhere update the tenant identifier
+  baseApiUrl: 'https://demo.openmf.org',  // For connecting to server running elsewhere update the base API URL
```

Comment:
Avoid making any changes to the environment.prod.ts file or any env based files.